### ISSUE TITLE
Debuff provider review

### ISSUE THREAD

**Issue by Adirelle** [AdiAddons/DRData-1.0#1]
Review the providers of debuffs for the following split by class, and racials.
- [ ] Death Knight
- [ ] Druid
- [ ] Mage
- [x] Hunter
- [x] Monk
- [ ] Paladin
- [ ] Priest
- [ ] Shaman
- [x] Warlock
- [ ] Warrior
- [ ] Racials

Hunter spells are partially done, pet spells still need to be checked.

**Comment by Adirelle** [comment]
Related commits and PR : https://github.com/Adirelle/DRData-1.0/pull/2 https://github.com/Adirelle/DRData-1.0/pull/3 https://github.com/Adirelle/DRData-1.0/commit/be7fe70e8411b3bffd2290e48f3729ef0c2d5a47

**Comment by Rainrider** [comment]
I have made a complete list for the DR categories in WoD beta, with providers and everything and even started with LPS (you could take a look [here](https://github.com/Rainrider/LibPlayerSpells-1.0/tree/wod). I'm not sure why we split data accross two libraries. DRData is very small by itself and could be merged in LPS. What is your view on that one?

**Comment by Adirelle** [comment]
I didn't wrote DRData initially, even though I maintaining it right now. 

I used it in ABA at first because it was easier to do so than to import all spells into LPS. However, I would ideally separate them. If we added the right metadata (i.e. the DR category of the debuffs) to LPS, it could implement DRData interface so it could work as drop-in replacement for it.

I'm wondering if how many addons still use DRData.
